estuary-cdk/estuary_cdk/http.py

```diff
@@ -125,31 +103,20 @@ async def request_lines(
 
         return
     
-    async def request_object_stream(
+    async def request_stream(
```

Comment:
Having this functionality tied to `http` felt a little cumbersome, and I guess you could end up with a byte stream in some other way, so I separated the JSON processing from the byte stream requesting.